CI_YAML.md

```diff
@@ -42,7 +42,10 @@ targets:
 # presubmit: Whether to run this target on presubmit (defaults to true).
 # postsubmit: Whether to run this target on postsubmit (defaults to true).
 # run_if: List of path regexes that can trigger this target on presubmit.
-#         If none are passed, will always run in presubmit.
+#         If none are passed, it will evaluare run_if_not. If both are empty the target
+#         will always run in presubmit.
+# run_if_not: List of path regexes. The target will be run only if the files changed
+#         do not match any paths in this list.
```

Comment:
What happens if someone supplies both of these properties?

It looks as if both are supplied it will ignore the run_if_not in that case. Someone will try this for sure; is this desired behavior?

Reply:
Documented that if both are provided and non empty run_if_not will be ignored.